appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -521,10 +535,12 @@ private void setScreenOrientationProperty(String text) {
         screenWidth = LANDSCAPE_WIDTH;
         screenHeight = LANDSCAPE_HEIGHT;
         landscape = true;
+        phoneWidget.setStyleName("ode-SimpleMockFormPhoneLandscape");
```

Comment:
Need logic similar to changePreviewSize. If you select Monitor sizing, and then switch to landscape, the image that shows is the phone image scaled way up.

Reply:
Code update pushed